WinUIGallery/ControlPages/XamlFundamentals/TemplatesPage.xaml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="WinUIGallery.ControlPages.TemplatesPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:WinUIGallery"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Page.Resources>
+        <ItemsPanelTemplate x:Key="WrapGridTemplate">
+            <WrapGrid Orientation="Horizontal" />
+        </ItemsPanelTemplate>
+
+        <ItemsPanelTemplate x:Key="StackPanelTemplate">
+            <StackPanel Orientation="Vertical" />
+        </ItemsPanelTemplate>
+    </Page.Resources>
+
+    <StackPanel>
+        <StackPanel Spacing="12"
+                    Margin="0,12,0,0">
+            <RichTextBlock>
+                <Paragraph FontWeight="SemiBold">Where can you create templates?</Paragraph>
+                <Paragraph>You can define templates at the app, page, or control level, just like styles and resources. The placement depends on where and how you want the template to be reused.</Paragraph>
+            </RichTextBlock>
+
+            <RichTextBlock>
+                <Paragraph FontWeight="SemiBold">Key points:</Paragraph>
+                <Paragraph>- Templates allow you to redefine how a control is built visually while keeping its behavior intact.</Paragraph>
+                <Paragraph>- Templates can be defined at the App, Page, or Control level, depending on the desired scope.</Paragraph>
+                <Paragraph>- Types of templates:</Paragraph>
+                <Paragraph>
+                    <Run Text="    -" />
+                    <Run FontWeight="SemiBold"
+                         Text="ControlTemplate" />
+                    <Run Text="customizes the structure of a control." />
+                </Paragraph>
+                <Paragraph>
+                    <Run Text="    -" />
+                    <Run FontWeight="SemiBold"
+                         Text="ItemsPanelTemplate" />
```

Comment:
I'm wondering if we should include `ItemsPanelTemplate`.. it is very specific. Thoughts?

Reply:
I noticed that `ItemsPanelTemplate` is used frequently in various control examples throughout the app, such as:  
- FlipView Page: 3rd example  
- GridView Page: 2nd example  
- ListView Page: 3rd and 4th examples  
- SplitButton Page: 1st example
- and it is utilized numerous times within the app itself

Given its repeated use in the app, I decided to include it.